lib/.gitignore

```diff
@@ -7,4 +7,4 @@
 **/*.js.map
 
 # Do not include .js files from .ts files
-dialects\index.js
+dialects/index.js
```

Comment:
this is done automatically by one of the npm scripts, in fact I can't even get rid of this change cause it gets added back every time I revert it